### PR TITLE
Spring Intro Task

### PR DESCRIPTION
**Pull request by IvanLuchkin**
- AppConfig created
- Main class added for testing
- User created, UserDao and UserService implemented
- pom.xml updated
- .gitignore updated

### CODE REVIEW

**Review by Mandybur800 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.idea/
+spring-intro.iml
+target/
```
Comment (dmytro-hryhoruk) [inline .gitignore:3]: empty line

File: src/main/java/spring/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import spring.model.User;
+
+@Repository
+public class UserDaoImpl implements UserDao {
+    @Autowired
+    private SessionFactory sessionFactory;
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/dao/UserDaoImpl.java:14]: let's use constructor autowiring

File: src/main/java/spring/service/UserServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import spring.dao.UserDao;
+import spring.model.User;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Autowired
+    private UserDao userDao;
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/service/UserServiceImpl.java:12]: same

File: src/main/java/spring/dao/UserDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+    @Override
+    public void insert(User user) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/spring/dao/UserDaoImpl.java:17]:
```suggestion
    public void add(User user) {
```

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]
I don't see travis.yml file here

File: src/main/java/spring/dao/UserDaoImpl.java (hunk 4 above)
Comment (dmytro-hryhoruk) [reply src/main/java/spring/dao/UserDaoImpl.java:21]: either fix the comment or answer on it please)

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/spring/dao/UserDaoImpl.java (hunk 4 above)
Comment (IvanLuchkin) [reply src/main/java/spring/dao/UserDaoImpl.java:21]: Sorry, I've lost it. Fixed

**Pull request merged**